src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1392,14 +1392,26 @@
 				}
 			}
 
-			prototype._buildArcListviewElement = function (page, cssClass) {
+			prototype._buildArcListviewElement = function (parentElement, cssClass) {
 				// find or add element for current list element
-				var arcListviewElement = page.querySelector(cssClass);
+				var arcListviewElement = parentElement.querySelector(cssClass);
 
 				if (!arcListviewElement) {
 					arcListviewElement = document.createElement("div");
 					arcListviewElement.classList.add(cssClass);
-					page.appendChild(arcListviewElement);
+					parentElement.appendChild(arcListviewElement);
+				}
+				return arcListviewElement;
+			};
+
+			prototype._buildArcListviewElementBefore = function (parentElement, cssClass) {
```

Comment:
This method looks quite similar to _buildArcListviewElement. How about having one method where it's possible to specify insertion method, for example:

`_buildArcListviewElement(parentElement, cssClass, insertionOption) `

where insertionOption could be
```
{ insertBefore: true }
```

Reply:
You are right.